packages/generator/src/parser/parser-util.ts

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+export function stripNamespace(name: string): string {
+  const nameParts = name.split('.');
```

Comment:
Here an exception would be good so that we realize if something goes wrong and the input is not: `myNamesape.myName`

Reply:
As long as `name` is a string this won't fail.

Reply:
true, but give a unexpected result. For a exported method I would like to have input validation. Even if it is that simple.

Reply:
Hmm, it isn't unexpected. It gives you everything after the last dot in the name. That means that if there is no dot, it just returns the input. I added some explicit tests where this becomes visible. Is there anything other way we can make this more clear?
Or do you mean you want to test it for inputs other than string?
(Btw, this has also been here for a long time, I just moved it to a separate file and used it consistently, where it was implemented redundantly.)

Reply:
I think I mean more, that if a value is passed inside not of the form `NameSpace.Name` something is wrong in the EDMX file. In general a namespace should be there and no dot in the name or name space is allowed. The method will handle it gracefully and return just the full thing or in case of WordA.WordB.WordC the WordC. However, I would like to get a stop of the generation. Since it is not clear that WordC is the name or that the generator works if there is no namespace at all.

Reply:
A namespace can contain multiple dots (e. g. in the reference services this is the case). I am not sure whether it is normally possible that a name can have no namespace. If this is normally "forbidden" it would make sense to add a warning or even an error, but I would probably even do this somewhere else...